ts/types/pagopa.ts

```diff
@@ -223,6 +225,10 @@ export type CreditCardPaymentMethod = RawCreditCardPaymentMethod &
   PaymentMethodRepresentation &
   WithAbi;
 
+export type PrivativePaymentMethod = CreditCardPaymentMethod & {
+  cardLogo?: string;
```

Comment:
What do you think about renaming  `cardLogo` to `gdoLogo`, in order to specify the image representation contained in the field, particular for this payment method?
Also what you think about using the type `ImageSourcePropType`, in order to have a safe image to render inside the component?

Reply:
Modified in [eb2435d](https://github.com/pagopa/io-app/pull/2878/commits/eb2435d07b01aa4954ac1e25301e05b83443df46)